### PR TITLE
Quiz app made.

### PR DESCRIPTION
**Pull request by TJain02**

### CODE REVIEW

File: CSS_Components/styles.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Creepster&family=Montserrat&display=swap');
+
+body {
+    background-color: rgba(0, 0, 0, 0.938);
```
Comment (DevNirah7) [inline CSS_Components/styles.css:4]: Can use color variable here

File: CSS_Components/styles.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    border-radius: 10px;
+}
+
+.quiz-card {
+    width: 20rem;
+    border: 2px solid red;
+    border-radius: 10px;
+    margin: 3rem;
+    background-color: rgba(169, 169, 169, 0.048);
+    box-shadow: 0 0.325rem rgba(169, 169, 169, 0.096);
```
Comment (DevNirah7) [inline CSS_Components/styles.css:49]: Can use variables here too

File: conjuring-rules.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                <button class=" play-btn">
+                    Start
+                </button>
+            </a>
+        </div>
+            
+        </ul>
+        
+     </div>
+
```
Comment (DevNirah7) [inline conjuring-rules.html]: Can beautify file and remove unwanted spacing

**Review by DevNirah7 — APPROVED** [review]
Variables as suggested and beautifying pages needed. Rest all good to go!

File: it-score.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+            Oops! Wrong answer. Score : 0
+        </div>
+        
+        <div class="total-score">
+            Total Score: 30
+        </div>
+
+    </div>
+    </div>
```
Comment (Asypher00) [inline it-score.html:121]: You have given the same answers to the questions of the conjuring quiz in it quiz

**Review by Asypher00 — COMMENTED** [review]
Suggested a change. Otherwise code looks good

File: conjuring-rules.html (hunk 3 above)
Comment (TJain02) [reply conjuring-rules.html]: Done.

**Pull request merged**